orbit/pkg/execuser/execuser_linux.go

```diff
@@ -119,10 +121,10 @@ func runWithStdin(path string, opts eopts) (io.WriteCloser, error) {
 	return stdin, nil
 }
 
-func getUserAndDisplayArgs(path string, opts eopts) ([]string, error) {
+func getUserAndDisplayArgs(path string) ([]string, []string, *string, error) {
```

Comment:
Nit, usually for readability I name the returns:
```go
func getUserAndDisplayArgs(path string) (args []string, env []string, userContext *string, err error) {
```

PS: Maybe rename to `getConfigurationForRunuser`? or somthing that includes the `runuser` word.